### PR TITLE
fix: update Expense Claim

### PR DESCRIPTION
**Pull request by barredterra**
Update docs on expense claim in German and English.

### CODE REVIEW

**Comment by barredterra** [comment]
Spell checker doesn't work for German.

**Comment by surajshetty3416** [comment]
@barredterra German docs are not checked for spelling errors. Spell check is failing because there's German word in English docs.

File: erpnext_com/www/docs/user/manual/en/human-resources/expense-claim.md [hunk 1]
```diff
@@ -62,21 +62,23 @@ User can view unpaid expense claim using report "Unclaimed Expense Claims"
 
 ### Payment for Expense Claim
 
-To make payment against the expense claim, user has to click on Make > Bank Entry
+To make payment against the expense claim, user has to click on Create > Payment.
+
 #### Expense Claim
-<img class="screenshot" alt="Expense Claim" src="{{docs_base_url}}/assets/img/human-resources/payment.png">
+
+<img class="screenshot" alt="Aufwandsabrechnung" src="{{ docs_base_url }}/assets/img/human-resources/expense_claim_create_payment.png">
```
Comment (surajshetty3416) [inline erpnext_com/www/docs/user/manual/en/human-resources/expense-claim.md]:
```suggestion
<img class="screenshot" alt="Create Payment" src="{{ docs_base_url }}/assets/img/human-resources/expense_claim_create_payment.png">
```

**Comment by barredterra** [comment]
Thanks for the fix!

**Pull request merged**